docs/DeveloperGuide.md

```diff
@@ -374,11 +371,79 @@ otherwise)
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. ClickConnect shows an error message.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**System**: ClickConnect
+
+**Use case**: UC02 - See usage instructions
+
+**Actor**: User
+
+**MSS**
+
+1. User opens ClickConnect application
```

Comment:
Maybe we can omit this information for all use cases since it is rather trivial? The application has to be open for any user interactions

Reply:
makes sense, ommited this step from all use cases, thanks!